{"uuid": "e645f354-18dc-4f8a-a118-96db794056e9", "children": ["171afa3f-384e-4f8f-9d3f-89874937e305"], "befores": [{"name": "xunit_setup_class_fixture_Test_verify_data", "status": "passed", "start": 1629720510081, "stop": 1629720525212}], "afters": [{"name": "xunit_setup_class_fixture_Test_verify_data::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\_pytest\\python.py\", line 815, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\_pytest\\python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"d:\\code\\tylerhub\\demo\\registration_process\\register_actions_bin\\test_verify_data_bin.py\", line 46, in teardown_class\n    verify.quitbrowser()\n  File \"d:\\code\\tylerhub\\demo\\registration_process\\register_positioning\\verify_message_location.py\", line 151, in quitbrowser\n    common.quit_browser()\n  File \"d:\\code\\tylerhub\\demo\\public\\browser_actions.py\", line 508, in quit_browser\n    self.driver.quit()\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 153, in quit\n    RemoteWebDriver.quit(self)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\request.py\", line 75, in request\n    return self.request_encode_url(\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\request.py\", line 97, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\poolmanager.py\", line 330, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 665, in urlopen\n    httplib_response = self._make_request(\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 387, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\http\\client.py\", line 1230, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\http\\client.py\", line 1276, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\http\\client.py\", line 1225, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\http\\client.py\", line 1004, in _send_output\n    self.send(msg)\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\http\\client.py\", line 944, in send\n    self.connect()\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\connection.py\", line 184, in connect\n    conn = self._new_conn()\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\connection.py\", line 156, in _new_conn\n    conn = connection.create_connection(\n  File \"C:\\Users\\MSI-NB\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\urllib3\\util\\connection.py\", line 74, in create_connection\n    sock.connect(sa)\n"}, "start": 1629720560990, "stop": 1629720567083}], "start": 1629720510081, "stop": 1629720567083}